mininterface/tk_interface/adaptor.py

```diff
@@ -108,13 +108,22 @@ def run_dialog(self, form: TagDict, title: str = "", submit: bool | str = True)
 
         # Set the submit and exit settings
         if self.form.button:
-            tip, keysym = ("Enter", "<Return>")
+            tip = "Enter (when button focused)"
             ToolTip(self.form.button, msg=tip)  # NOTE is not destroyed in _clear
-            self._bind_event(keysym, self._ok)
 
-            # submit button styling
```

Comment:
You may restore the whole file. This is an alternate submit button styling that we might you in the future, please keep it :)